mysql/server/database.sls

```diff
@@ -1,11 +1,14 @@
-{%- from "mysql/map.jinja" import server with context %}
+{%- from "mysql/map.jinja" import server, mysql_connection_args with context %}
 
 {%- if not grains.get('noservices', False) %}
 {%- for database_name, database in server.get('database', {}).iteritems() %}
 
 mysql_database_{{ database_name }}:
   mysql_database.present:
   - name: {{ database_name }}
+  - connection_user: {{ mysql_connection_args.user }}
+  - connection_pass: {{ mysql_connection_args.password }}
+  - connection_args: {{ mysql_connection_args.charset }}
```

Comment:
Yeah sorry, I fixed that. I don't have to push things this early in the morning :)